src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -1165,6 +1165,24 @@ CrontabSchedule VerifyBackupFrequency(string backupFrequency)
             }
         }
 
+        [RavenAction("/admin/unused-databases", "POST", AuthorizationStatus.Operator)]
+        public async Task SetUnusedDatabaseIds()
+        {
+            var database = GetStringQueryString("database");
```

Comment:
change to `name` to match other endpoints

Reply:
done